src/librustdoc/clean/types.rs

```diff
@@ -1856,12 +1856,12 @@ impl Span {
         Self(sp.source_callsite())
     }
 
-    crate fn dummy() -> Self {
-        Self(rustc_span::DUMMY_SP)
+    crate fn inner(&self) -> rustc_span::Span {
+        self.0
     }
 
-    crate fn span(&self) -> rustc_span::Span {
-        self.0
+    crate fn dummy() -> Self {
+        Self(rustc_span::DUMMY_SP)
     }
```

Comment:
Why reorder the functions?

Reply:
I did it to put `dummy` next to `is_dummy` and `from_rustc_span` next to `inner` because those pairs correspond. But I can also see an argument for having all the constructors first. Let me know which you prefer.

Reply:
Sure, that seems reasonable.